Modules/core/Controller/CoreusersController.php

```diff
@@ -117,7 +119,7 @@ public function editAction($id) {
         }
         $script = "";
         if ($form->check()) {
-            if ($modelUser->isLogin($this->request->getParameter('login'))) {
+            if ($modelUser->isAnotherUserLogin($this->request->getParameter('login'), $user["login"])) {
```

Comment:
why calling an other fonction here?
We just want to differentiate "create new user" vs "edit existing user". If id==0, then it is a create, else an edit

Reply:
That's because the bug avoiding to validate modifications on user was linked to this function. In fact, CoreUser::isLogin($login) method returns true, and then cancels user modification, if the login already exists in database. Obviously, when editing a user (and not creating a new one), it is always true. So I choose to create a new function, working in both cases, taking into account the login of the user we want to edit. (cf CoreUser::isAnotherUserLogin($login, $actualUserLogin)). Didn't know how to name this function more clearly.